de.wwu.md2.framework/src/de/wwu/md2/framework/generator/mapapps/AppClass.xtend

```diff
@@ -60,6 +63,12 @@ class AppClass {
     
     // bundles section used to overwrite or add any property defined by components in the manifest.json of bundles
     "bundles": {
+        "md2_workflow_store": {
+            "MD2WorkflowStore": {
+                    "uri": "«dataContainer.main.defaultConnection.uri»workflowState/",
```

Comment:
I think it would be helpful to ensure that the defaultConnection URI has a trailing slash. We can either do that in the preprocessor (which I'd prefer) or alternatively directly in the DSL?